.circleci/config.yml

```diff
@@ -45,7 +45,6 @@ commands:
           command: |
             sudo apt update
             sudo apt install --assume-yes python3-pip
-            sudo ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
Added in #3338 due to change in the node executor. But looks like this is no longer needed with the `22.5` version.